### ISSUE TITLE
Throw exception for invalid cache service ID

### ISSUE THREAD

**Issue by Najki** [liip/LiipTestFixturesBundle#67]
**Is your feature request related to a problem? Please describe.**

I took part lately in upgrading a project to newer version of Symfony which also included upgrading the version of LiipFunctionalTestBundle and usage of newer LiipTestFixturesBundle. Our tests started to suffer from really bad performance. It turned out that we've made a typo in config Yaml for parameter `liip_test_fixtures.cache_db.sqlite`. The `AbstractDatabaseTool::getBackupService()` method loads the service only if it exists and silently disables all cache when it does not exist. So our project stopped using fixtures cache and we weren't aware of that up until we investigated the problem with Xdebug.

**Describe the solution you'd like**

I would like to know when there is an ID of non-existent service used in my configuration. I would expect the cache warmup or tests to raise an exception in this case.

**Comment by alexislefebvre** [comment]
Thanks for reporting this. What was the typo?

**Comment by Najki** [comment]
We've used `liip_functional_test.services_database_backup.sqlite` instead of `liip_test_fixtures.services_database_backup.sqlite`, but I don't think it's relevant. What I've meant is that it would be nice if LiipTestFixturesBundle would make our tests to fail when we use a non-existent service ID for the `liip_test_fixtures.cache_db.sqlite` configuration setting.

**Comment by alexislefebvre** [comment]
Thanks, I understand.

There's something to do here to stop failing silently: https://github.com/liip/LiipTestFixturesBundle/blob/52ddcd2db816e1826ac14e114332795fe75cb747/src/Services/DatabaseTools/AbstractDatabaseTool.php#L125-L130

**Comment by Najki** [comment]
Exactly. Maybe something like this?

```php
if ($this->container->hasParameter($backupServiceParamName)) {
    $this->container->get(
        $this->container->getParameter($backupServiceParamName)
    );
}
```

**Comment by alexislefebvre** [comment]
It should be fixed by #78
